setup.py

```diff
@@ -27,7 +27,7 @@ def local_scheme(version):
     use_scm_version={"local_scheme": local_scheme},
     setup_requires=["setuptools_scm"],
     install_requires=["setuptools", "wcwidth"],
-    extras_require={"tests": ["pytest", "pytest-cov"]},
+    extras_require={"tests": ["pytest", "pytest-cov"], "windows": ["colorama"]},
```

Comment:
I think we could call this `color`, because even though colorama is only needed for Windows, it's only needed for colour and not all Windows installs.

```suggestion
    extras_require={"tests": ["pytest", "pytest-cov"], "color": ["colorama"]},
```

In fact, let's make `color` also depend on Windows, something like:

https://discuss.python.org/t/can-extras-require-values-be-os-dependant/2718/2?u=hugovk